systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -151,4 +154,75 @@ public static void waitForClusterStability(String clusterName) {
             return false;
         });
     }
+
+    /**
+     * Method which, update/replace Kafka configuration
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static void updateSpecificConfiguration(String clusterName, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+        KafkaResource.replaceKafkaResource(clusterName, kafka -> {
+            LOGGER.info("Kafka config before updating '{}'", kafka.getSpec().getKafka().getConfig().toString());
```

Comment:
How the output of this log looks? It's formated to be easily readable ?

Reply:
Yes :)